packages/core/src/components/field-label/field-label.scss

```diff
@@ -0,0 +1,19 @@
+@use '../../helpers';
+
+@mixin FieldLabel() {
+  .ods-field-label {
+    @include helpers.font('300-regular');
+
+    color: helpers.color('content-main');
+    display: flex;
+    flex-flow: column wrap;
+    padding-top: helpers.space(0.5);
```

Comment:
The example below will incorrectly have 4px padding top:

```tsx
<label class="ods-field-label">
  <input class="ods-input" />
</label>
```

Do we just not support that usage? Or don't care about that "error"?

Could be fixed with `gap` but then this one would be the issue:

```tsx
<label class="ods-field-label">
  My label
  <span class="ods-helper-text">Some help</span>
  <input class="ods-input" />
</label>
```

Personally, I guess the first example having 4px padding top is the least problematic, `<label>` with no copy/text is quite unusual.

Reply:
4px at the top is by design.

Design also requires to always follow 8px grid. So this then gets a little tricky.

Your example below has 48px of input size + 4px padding of a label, then 4px margin for an input, so 56px.

```html
<label class="ods-field-label">
  <input class="ods-input" />
</label>
```

This example also adds a 24px text (but still keeps 8px grid).

```html
<label class="ods-field-label">
  My label
  <input class="ods-input" />
</label>
```

This example below also adds a 24px helper text, so in the end is 104px - still 8px grid.

```html
<label class="ods-field-label">
  My label
  <span class="ods-helper-text">Some help</span>
  <input class="ods-input" />
</label>
```

Do you see any potential issue here?

Reply:
Ah right, it'll be a 8px total of empty space above the `<input />`.

I mean, you shouldn't use `<label />` if you're not going to have text inside it, right?
The empty space by spec should not be there if there's not copy/text for a label, but otherwise we complicate the usage by a lot.

I guess we just say this is incorrect usage:
```tsx
<label class="ods-field-label">
  <input class="ods-input" />
</label>
```

Either include copy/text for the label or remove the `<label />` element altogether.
Agree?

Reply:
Yes.

Let's point this out to designers (if they will understand what's the issue here).

At the end there will still be additional docs provided, e.g. on how to set required/optional fields.

Would you suggest to include anything in current docs, if so what and where?

Reply:
Hmm good question. For React probably only on FieldLabel's story, as `Input` and `Textarea` should receive the label as children and not worry about `FieldLabel` itself.

For HTML stories we'll have to be more thorough.